src/components/PoolOverview.tsx

```diff
@@ -35,6 +35,15 @@ function PoolOverview({
           }
         : acc
     }, {} as Partial<Record<Partners, string>>),
```

Comment:
This isn't feedback on your code, but could you change the `reserve` formatting to `commify(formatBNToString(poolData.reserve, 18, 0))` to get rid of the cents? Makes it cleaner :)